org.godotengine.Godot.yaml

```
@@ -3,6 +3,7 @@ runtime: org.freedesktop.Platform
 runtime-version: '21.08'
 default-branch: stable
 sdk: org.freedesktop.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.openjdk11]
```

Prefer full array syntax for consistency with the rest of the manifest:

```suggestion
sdk-extensions: 
  - org.freedesktop.Sdk.Extension.openjdk11
```